chapters/annotations.tex

```diff
@@ -384,19 +384,23 @@ \section{Annotations for Simulations}\label{annotations-for-simulations}
 
 \begin{annotationdefinition}[experiment]
 % henrikt-ma 2021-04: Seems strange to allow 'experiment' completely without list of options -- what would it mean?
+
```

Comment:
Since the PR states that the purpose is to no longer allow the argument-less `experiment` annotation, this comment should be removed as part of the PR:
```suggestion
```

Also, I'm afraid a paragraph break here could confuse the layout engine of pdfLaTeX or LaTeXML.